etc/c/odbc/sql.d

```diff
@@ -67,17 +67,17 @@ enum
     SQL_HANDLE_DESC                 = 4,
 
     // * environment attribute *
-    SQL_ATTR_OUTPUT_NTS             = 10001,
+    SQL_ATTR_OUTPUT_NTS             = 10_001,
```

Comment:
Sorry, I thought that was a non controversial Dscanner setting.
The next PR related to Dscanner or updating these bindings could simply remove `etc` [from the `find` in `posix.mak`](https://github.com/dlang/phobos/blob/master/posix.mak#L530